### PR TITLE
Use $themeConfig.searchPlaceholder for search placeholder

### PR DESCRIPTION
**Pull request by alexandernst**

### CODE REVIEW

**Comment by alexandernst** [comment]
@z3by should I attach a screenshot here as well?

File: components/SearchBox.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
       v-model="query"
       size="small"
       :fetch-suggestions="querySearchAsync"
-      placeholder="Search"
+      :placeholder="$themeConfig.searchPlaceholder"
```
Comment (z3by) [inline components/SearchBox.vue:9]: We need to provide a default value for the place holder, what if we did not specify `$themeConfig.searchPlaceholder` in our config file?

**Comment by alexandernst** [comment]
It seems that this value is provided by vuepress: https://vuepress.vuejs.org/theme/default-theme-config.html#search-placeholder

It should have a default value, but I can add a default “Search” text if you want me to do it.

**Comment by alexandernst** [comment]
Ok, I just tested it and there is no default value, so I'll add one.

**Comment by alexandernst** [comment]
@z3by Done ⬆️

**Comment by z3by** [comment]
Thank you! yeah, it is used by the default search plugin used by Vuepress, but we are not using it here.
I really appreciate your contributions @alexandernst it makes so much difference in the quality of this theme!

**Pull request merged**